_docs_integrate/25-requests-over-templates.md

```diff
@@ -0,0 +1,158 @@
+For simplicity the Request inside the Template only contains a AuthenticationRequestItem, but you can use any RequestItems you want.
+
+```json
+{
+    "content": {
+        "items": [
+            {
+                "@type": "AuthenticationRequestItem",
+                "mustBeAccepted": true,
+                "title": "The templator is asking for an authentication"
+            }
+        ]
+    }
+}
+```
+
+## Create the Template
+
+Create the Relationship Template on the Templators Connector. You can do so by calling the `POST /api/v2/RelationshipTemplates/Own` route. Use the following JSON in the Request body:
+
+```jsonc
+{
+    "maxNumberOfAllocations": 1,
+    "expiresAt": "2023-06-01T00:00:00.000Z",
+    "content": {
+        "@type": "RelationshipTemplateContent",
+        "title": "Connector Demo Contact",
+        "onNewRelationship": {
+            // <the content validated in the step before>
+        }
+    }
+}
+```
+
+You will receive a response with the complete RelationshipTemplate. In this guide we will only use the Templates truncated reference and its id. Thus make sure to save the `truncatedReference` and the `id` for the next steps.
+
+```jsonc
+{
+    "id": "RLT..",
+    // ...
+    "truncatedReference": "UkxU..."
+}
+```
+
+## Load the Template on the Requestor Connector
+
+Now you have to load the Template on the Requestor Connector. You can do so by calling the `POST /api/v2/RelationshipTemplates/Peer` route with the following content. Use the truncated reference you copied before:
+
+```jsonc
+{
+    "reference": "UkxU..."
+}
+```
+
+If no Relationship exists this will trigger a process in the Request module. The Request module will create a new incoming Request on that we will work in the next step. You can observe this by long polling the incoming Requests or by waiting for the `consumption.incomingRequestReceived` event.
+
+The long polling is done by calling the `GET /api/v2/Requests/Incoming` route. You can use the query params `source.reference=<id-of-the-template>` and `status=ManualDecisionRequired` to filter only for Requests that belong to the Template you are currently working on.
+
+For more information about the events you can head over to the [Connector Modules site]({% link _docs_integrate/03-connector-modules.md %}) and read about the `AMQP Publisher` and the `WebhooksV2` module that are handling events.
+
+If you got the Request save the `id` of the Request for the next step.
+
+## Answer the Request
+
+The rejection is explained before the acceptance because you can re-do it as often if you want. If you accept the Request a RelationshipRequest will be sent and no new Request will be created until the RelationshipRequest is answered. If it is accepted the Request module will recognize the existing Relationship and will also not create a new Request.
+
+We therefore recommend to first reject the Request and then accept it if you want to test the full flow.
```

Comment:
Ich würd das "We" hier weglassen und eher im Passiv schreiben. Also sowas wie "So if you want to test the full flow, you should first reject the Request. After that you can create a new one, which you can accept."